README.md

```diff
@@ -2,14 +2,43 @@ _NOTE: The interfaces of flux-accounting are being actively developed and are no
 
 ## flux-accounting
 
-Development for a bank/accounting interface for the Flux resource manager. Writes and saves job step history, user account information, and priority calculation to persistent storage using Python's SQLite3 API. 
+Development for a bank/accounting interface for the Flux resource manager. Writes and saves user account information to persistent storage using Python's SQLite3 API.
 
-##### job step history
+### Build Requirements
 
-Running as a cron job, `write_jobs.py` uses a flux-core RPC to retrieve job data that have recently transitioned to an INACTIVE state in the past _x_ minutes. It parses the job record data and writes it to a SQLite database, which can be queried at a later time. 
+flux-accounting requires the following packages to build:
 
-##### user account information
+| centos8       | ubuntu      | version |
+| ------        | --------    | ------- |
+| python3-devel | python3-dev | >= 3.6  |
+| python3-pip   | python3-pip | 20.0.2  |
+| tox           | tox         | 3.15.0  |
+
+### Install Instructions
+
+You can install the dependencies required by flux-accounting (located in **requirements.txt**) with the following command:
+
+```
+$ pip3 install -r requirements.txt
+```
+
+### Test Instructions
 
-Users have banks that they can charge their jobs against; this will affect their priority when submitting future jobs in order to maintain a fair balance of resources between users running jobs on a single cluster. Multiple factors play a role in determining a user's job priority, but in general, the amount of resources used vs. the resource limits initially allocated will either lower or heighten a user's job priority. 
+Make sure all is right correctly by running the unit tests with `tox`:
```

Comment:
< Make sure all is right correctly by running the unit tests with `tox`:

> Run the unit tests with `tox` to ensure the correctness of this package on your platform:

Reply:
Thanks @dongahn 👍 I went ahead and pushed a change to reflect your suggestion.